### PR TITLE
4388 - CourtOrderPoa - New prop to make courtOrderNumber required regardless PoA

### PR DESCRIPTION
**Pull request by andrepestana-aot**
*Issue #:* /bcgov/entity#4388

*Description of changes:*

- Added new prop to make courtOrderNumber required regardless PoA
- Update tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/CourtOrderPoa/CourtOrderPoa.vue [hunk 1]
```diff
@@ -62,6 +62,10 @@ export default class CourtOrderPoa extends Vue {
   @Prop({ default: true })
   readonly displaySideLabels: boolean
 
+  /** Wether court order number is required regardless plan of arrangement. */
+  @Prop({ default: false })
+  readonly courtOrderNumberRequired: boolean
```
Comment (severinbeauvais) [inline src/components/CourtOrderPoa/CourtOrderPoa.vue:67]: Under what conditions is Court Order Number required?

Is this the validation for https://github.com/bcgov/entity/issues/7473  ?

Comment (andrepestana-aot) [reply src/components/CourtOrderPoa/CourtOrderPoa.vue:67]: No, it's for bcgov/entity#4388. The court order notation should have should have the court order number required. This was confirmed today by Linda.

Comment (severinbeauvais) [reply src/components/CourtOrderPoa/CourtOrderPoa.vue:67]: ok

Comment (andrepestana-aot) [reply src/components/CourtOrderPoa/CourtOrderPoa.vue:67]: Does "ok" mean "approved"? 😄

**Review by cameron-freshworks — APPROVED** [review]
Looks good to me!
You might need to do a quick rebase and rebuild of the Shared Components to resolve the conflict. 
Another small pr was just merged which most likely caused that, sorry about that :(

**Comment by andrepestana-aot** [comment]
> Looks good to me!
> You might need to do a quick rebase and rebuild of the Shared Components to resolve the conflict.
> Another small pr was just merged which most likely caused that, sorry about that :(

No problem, sir. Thanks!

**Pull request merged**